decidim-dev/lib/decidim/dev/test/rspec_support/warden.rb

```diff
@@ -1,7 +1,23 @@
 # frozen_string_literal: true
 
+module Decidim
+  module WardenTestHelpers
+    include Warden::Test::Helpers
+
+    #
+    # Utility method to login in the middle of a test as a different user from
+    # the current one.
+    #
+    def relogin_as(user, scope:)
```

Comment:
`scope` is supposed to have the default value `:user`, right? Also you need to use the same value for the `logout` function.